src/Upgrade/Application/Strategy/Common/IntegratorEvaluator.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+/**
+ * Copyright © 2016-present Spryker Systems GmbH. All rights reserved.
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+declare(strict_types=1);
+
+namespace Upgrade\Application\Strategy\Common;
+
+use Upgrade\Application\Provider\ConfigurationProviderInterface;
+
+class IntegratorEvaluator implements IntegratorEvaluatorInterface
```

Comment:
Since we have both products the name of the class might confuse. 

```suggestion
class IntegratorExecutionValidator implements IntegratorExecutionValidatorInterface
```

Reply:
+